### DISCUSSION TITLE
How to prevent ROB ending simulation when Dispatch is stalled by execution unit unavailability?

### DISCUSSION THREAD

**Discussion by egieske** [riscv-software-src/riscv-perf-model#133] [Q&A] [answered]
I am using the original blocking single-issue LSU model and the ROB ends simulation before the trace ends because it hasn't received new instructions because Dispatch is stalled due to a long LSU miss latency. I recall there was a discussion or issue about this, but haven't found it.

**Comment by klingaard** [comment, marked as answer]
Yeah, this is addressed in PR #91. I think it's ready for merge, so might try again after that.

**Comment by klingaard** [comment]
This is now merged.  Please let me know if you still see the issue.
